### PR TITLE
Add recipe for joommf.

### PR DESCRIPTION
**Pull request by marijanbeg**

### CODE REVIEW

File: recipes/joommf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py27]
+  skip: True  # [win32]
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: Rename this to `host:`

File: recipes/joommf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py27]
+  skip: True  # [win32]
+  number: 0
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: Add `noarch: python` here

File: recipes/joommf/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py27]
+  skip: True  # [win32]
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: `noarch: python` can't have selectors, so remove these and in requirements change `python` to `python >= 3.5`

File: recipes/joommf/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  skip: True  # [win32]
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: Remove this line

File: recipes/joommf/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  number: 1
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: Change this to 0

File: recipes/joommf/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  script: pip install --no-deps .
+
+requirements:
+  host:
+    - python >= 3.5
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: No space between `>=` and `3.5`

**Comment by marijanbeg** [comment]
Hi @isuruf, thanks a lot for reviews. The dependency of this package is OOMMF which is built only for x64, so we have to skip the x32 build.

File: recipes/joommf/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win32]
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  host:
+    - python>=3.5
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: Need space between `python` and `>=`

**Comment by isuruf** [comment]
Please keep the noarch: python. You can remove skip win32 line, because conda solver will not install this package on win32 as the dependency is not on win32

File: recipes/joommf/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  imports:
+    - joommf
+    - oommfc
+
+  commands:
+    - python -c "import joommfutil; joommfutil.test()"
+    - python -c "import discretisedfield; discretisedfield.test()"
+    - python -c "import oommfodt; oommfodt.test()"
+    - python -c "import micromagneticmodel; micromagneticmodel.test()"
+    - python -c "import oommfc; oommfc.test()"
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: Can you remove the 5 lines above? They shouldn't be in this package

**Comment by marijanbeg** [comment]
@isuruf , what should be the next step?

File: recipes/joommf/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+  doc_url: http://joommf.readthedocs.io
+  dev_url: https://github.com/joommf/joommf
+
+extra:
+  recipe-maintainers:
+    - marijanbeg
+    - fangohr
+    - takluyver
+    - davidcortesortuno
+    - rpep
```
Comment (isuruf) [inline recipes/joommf/meta.yaml]: @fangohr, @takluyver, @davidcortesortuno, @rpep, are you okay with being maintainers?

Comment (rpep) [reply recipes/joommf/meta.yaml]: Yes, fine with me

Comment (davidcortesortuno) [reply recipes/joommf/meta.yaml]: Yes

Comment (fangohr) [reply recipes/joommf/meta.yaml]: Yes, that's fine. Thank you!

**Pull request merged**